src/components/spreadsheet/utils/equipment-table-editors.js

```diff
@@ -100,7 +100,7 @@ export const NumericalField = forwardRef(
 
         const isValid = useCallback(
             (val, minVal, maxVal) => {
-                if (!val) {
+                if (val === undefined || val === null) {
```

Comment:
You can do that 
`if (val == null) {`
This will check for both undefined and null without needing two distinct checks